src/styles/getNavigationModalCardStyles/index.website.js

```diff
@@ -1,14 +1,12 @@
 import getBaseNavigationModalCardStyles from './getBaseNavigationModalCardStyles';
 
-export default ({windowHeight, isSmallScreenWidth}) => ({
+export default ({isSmallScreenWidth}) => ({
     ...getBaseNavigationModalCardStyles({isSmallScreenWidth}),
 
     // This height is passed from JavaScript, instead of using CSS expressions like "100%" or "100vh", to work around
```

Comment:
This comment is outdated, "from JavaScript" was referring to `${windowHeight}`.

Reply:
Comment has been upadated